Source/WebCore/dom/Document.cpp

```diff
@@ -3057,7 +3057,7 @@ void Document::collectHighlightRangesFromRegister(Vector<WeakPtr<HighlightRange>
     // One range can belong to multiple highlights so resetting a range's flag cannot be done in the loops above.
     for (auto& highlight : highlightRegistry.map()) {
         for (auto& highlightRange : highlight.value->highlightRanges()) {
-            if (auto* liveRange = dynamicDowncast<Range>(highlightRange->range()); liveRange && liveRange->didChangeForHighlight())
+            if (auto liveRange = RefPtr { dynamicDowncast<Range>(highlightRange->range()) }; liveRange && liveRange->didChangeForHighlight())
```

Comment:
I see this pattern a lot. Do people really prefer `auto liveRange = RefPtr { foo() }` over `RefPtr liveRange = foo()`?

Reply:
(I don't have a very strong opinion, but I usually opt for the latter)

Reply:
Well, it's auto liveRange = RefPtr further up in this file, so I was going for consistency. I think that in general, people really like `auto`, so that might be pushing things in that direction.

Reply:
people like auto... when it lets them type less/repeat themselves less, not just for fun. In this case the auto case is *more* typing and more redundant